Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalNoise_cff.py

```diff
@@ -3,7 +3,7 @@
 
 import HLTrigger.HLTfilters.hltHighLevel_cfi
 noiseHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
-    HLTPaths = ['HLT_MET25'],
+    HLTPaths = ['HLT_PFMET130'],
 #    eventSetupPathsKey='HcalCalNoise',
```

Comment:
This is not what we should do.... what we should do is the commented out line, i.e. have
`eventSetupPathsKey='HcalCalNoise'` active.

Reply:
Done